packages/server/graphql/types/AddMattermostAuthPayload.ts

```diff
@@ -0,0 +1,32 @@
+import {GraphQLObjectType} from 'graphql'
+import MattermostIntegration from './MattermostIntegration'
+import makeMutationPayload from './makeMutationPayload'
+import User from './User'
+import {GQLContext} from '../graphql'
+
+export const AddMattermostAuthSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'AddMattermostAuthSuccess',
+  fields: () => ({
+    MattermostIntegration: {
```

Comment:
hot take: let's keep this all lowercase. since we have `integration.mattermost`, don't wanna mix up camel & pascal